About.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <script src="https://kit.fontawesome.com/15d46053c4.js" crossorigin="anonymous"></script>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Lato:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <title>Auto-Conference</title>
+</head>
+
+<body>
+
+  <header>
+    <div id="topnav">
+      <a href=""><img src="./Assets/pngkey.com-facebook-logo-png-white-1532072.png" alt="Facebook" class="social"></a>
+      <a href=""><img src="./Assets/new-twitter.png" alt="Twitter" class="social"></a>
+      <ul id="topnavList">
+        <li>English</li>
+        <li>Home</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <nav class="dropDown">
+      <i class="fa-solid fa-bars" id="menuOpen"></i>
+      <ul id="menuList">
+        <a href="./Main.html#topnav" class="mlogo1">LAGF</a>
+        <i class="fa-solid fa-times" id="menuClose"></i>
+        <div class="droplist">
+          <li><a href="#about">About</a></li>
+          <li><a href="./Main.html#mainProgram" id=>Program</a></li>
+          <li><a href="./Main.html#bookBtn" id=>Join</a></li>
```

Comment:
- [ ] Great job working on the home menu. The about mobile menu should also do same to cover the page elements until it is closed, or a link is clicked. Please kindly fix this minor issue.
![Screenshot (1222)](https://github.com/Godedict/Auto-Conference/assets/65298048/d906ee04-60a1-40c2-8ea0-fe37d0a3da99)

Reply:
Done